src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -106,32 +107,39 @@ class SparqlQleverVisitor {
   void addVisibleVariable(string var);
   void addVisibleVariable(Variable var);
 
+  [[nodiscard]] SparqlExpressionPimpl visitExpressionPimpl(auto* ctx);
```

Comment:
Should have a small comment + be close to the other "helper" functions like `visitVector`, `visitAlternative`, etc.

Reply:
The reason why I placed it there is that I think that maybe the helper functions should all be private. They are not part of the visitors interface after all. Probably should discuss that in the meeting. But I agree that they should still be in one place in the header.